src/EnergyPlus/SingleDuct.hh

```diff
@@ -94,10 +94,6 @@ namespace SingleDuct {
     // DERIVED TYPE DEFINITIONS
 
     // MODULE VARIABLE DECLARATIONS:
-    extern Array1D<Real64> MassFlow1; // previous value of the terminal unit mass flow rate
-    extern Array1D<Real64> MassFlow2; // previous value of the previous value of the mass flow rate
-    extern Array1D<Real64> MassFlow3;
-    extern Array1D<Real64> MassFlowDiff;
     extern bool GetInputFlag;   // Flag set to make sure you get input once
```

Comment:
Moved as member variable to eliminate passing air terminal index as argument to member functions